puppet/modules/utility/manifests/init.pp

```diff
@@ -28,10 +28,7 @@
 
   package { "ruby-shadow":
     ensure => installed,
-    name => $osfamily ? {
-      Debian => "libshadow-ruby1.8",
```

Comment:
We've only got Wheezy slaves, and now one Ubuntu 14.10 (which I hope we can rebuild to 16.04 when it's out).